test/index.test.js

```diff
@@ -334,5 +356,68 @@ describe('module-resolver', () => {
                 transformerOpts,
             );
     });
+
+    describe('unknown filename', () => {
```

Comment:
Oh I see, it's in opposition of the parent test which defines `filename`. Gotcha. It's ok!